src/components/Chart/Bars.js

```diff
@@ -19,7 +19,8 @@ import {
   deduplicate,
   unsafeDatumFn,
   subsup,
-  getTextColor
+  getTextColor,
+  getLastItemFromArray
```

Comment:
`getLastItemFromArray`is a bit of a ~mouthful~ readful (plus it's a little misleading, you don't actually get anything back). I'd stick to `last` or `isLast`/ `isLastItem` is `last`is a little too terse for your taste.